controllers/app.controllers.js

```diff
@@ -43,10 +44,18 @@ exports.getArticleById = (req,res,next) => {
 
 exports.getCommentsByArticleId = ((req,res,next) => {
     const {article_id} = req.params
-    selectCommentsByArticleId(article_id)
-    .then(comments => {
-        res.status(200).send({comments})
+
+    // Check that the article exists using the article id
+    const promises = [checkArticleExists(article_id)]
+
+    // Find the comments array associated with the article id
+    promises.push(selectCommentsByArticleId(article_id))
+
```

Comment:
I like this, however, why do you need to push the 2nd promise and not have it in the array to begin with? like:

`const promises = [checkArticleExists(article_id), selectCommentsByArticleId(article_id)]`